### DISCUSSION TITLE
ignore arguments and flags after double dash

### DISCUSSION THREAD

**Discussion by malko** [goreleaser/goreleaser#3893] [Ideas, Issue Triage, and General discussions]
I'm wondering is there a reason for goreleaser to take into account args that are passed after a double hyphen.
It is common practice to ignore arguments after " -- " in ci tools. I'm using a tool that allow to pass additional arguments to multiple commands, but goreleaser complain about it. Typically this is my use case, I have a bunch of tasks launched by a task runner at the same time, one of thoose is goreleaser. Some tools need to take a --version=$version arguments as they don't detect it from git describe. 
For now the only way to achieve this is to split it to multiple tasks in my CI which is not a big deal, but if we could do: 
- goreleaser release -- --version=$version
then i would be able to chain it in the same run.

Is there a reason to keep thoose arguments read by goreleaser, or is there any interest in me making a PR to exclude thosse arguments ?

**Comment by caarlos0** [comment]
probably a bug, feel free to pr it

I guess no one stumbled over it yet because there aren't many reasons to use `--` with goreleaser, as it wouldn't pass it down anywhere anyway 🤔

**Comment by malko** [comment]
digging a litle bit further, it seems the problem is more a common cobra problem. So I think we should patch cobra first, will follow up.

**Reply by malko** [reply]
here's the PR to cobra which should help in resolving this https://github.com/spf13/cobra/pull/1941

**Reply by caarlos0** [reply]
awesome, thank you!
